### PR TITLE
add a python tool to help update cron entries in batch

### PR DESCRIPTION
**Pull request by jianlinliu**

### CODE REVIEW

**Comment by jianlinliu** [comment]
cc @yunjiang29 and @liangxia for review

**Comment by jianlinliu** [comment]
> @jianlinliu one problem is, if `openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml` is updated, then `openshift-openshift-tests-private-release-4.12-periodics.yaml` needs to be updated as well.

Right, I want to leave that to users to manually run `make jobs` to get update accordingly.

File: ci-operator/config/openshift/openshift-tests-private/tools/update-cron-entries.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        print("updating test job - %s" %(test_name))
+        cron_output = shell("generate-cron-entry.sh %s %s" %(test_name, target_file))["out"]
+        cron_entry = cron_output.split("cron:")[-1].strip()
+        print(cron_entry)
+        test['cron'] = cron_entry
+        new_tests_list.append(test)
+    
+    all_data['tests'] = new_tests_list
+    
+    backup_file = "%s.%s" %(target_file, datetime.now().strftime("%Y%m%d%H%M%S"))
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/tools/update-cron-entries.py:65]: do we really need this backup file? b/c everyone is using git.

**Comment by yunjiang29** [comment]
/lgtm

@jianlinliu just one minor comment, you can ignore it.

File: ci-operator/config/openshift/openshift-tests-private/tools/update-cron-entries.py (hunk 1 above)
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/tools/update-cron-entries.py:65]: I cam turn it off, but actually i leave it there to give users a chance to compare for safety.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/tools/update-cron-entries.py]: I did some update to introduce `--backup` option, users can disable it via `--backup=no`, also the tool can support to edit multiple yaml files in one line.

File: ci-operator/config/openshift/openshift-tests-private/tools/update-cron-entries.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            result["err"] = cmd_stderr.strip().decode()
+            result["out"] = cmd_stdout.strip().decode()
+        result["rc"] = res.returncode
+    except Exception as ex:
+        print("Error: %s" % str(ex))
+    return result
+
+def usage():
+    print ("""
+Usage: python %s --backup [yes|no] file1 file2 ...
```
[the remark this answers is not in the document]
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/tools/update-cron-entries.py]: Done

**Pull request merged**